content/general/concepts/what-is-the-builder-pattern/what-is-the-builder-pattern.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: 'What is the Builder Pattern?'
+Description: 'The builder pattern decouples the responsibility of object creation from the desired objects class.'
+Subjects:
+- 'Computer Science'
+- 'Interview Prep'
+  Tags:
+- 'Classes'
+- 'Conceptual'
+- 'Constructors'
+- 'Objects'
+  CatalogContent:
+- 'learn-java'
+- 'paths/computer-science'
+---
+The _builder pattern_ decouples the responsibility of object creation from the desired object's class. It is normally implemented when a class 
+requires many properties to be set during it's construction and may be used to ensure immutability. 
+
+## UML Design
+
+![UML diagram of a builder](media/builder-uml.png)
```

Comment:
Check this file. What is uploaded doesn't seem right. Just two headers and blank space.

Reply:
It looks okay on my end. The only thing I can think of is the two headers are filled in white where as the rest of the info is written in black on a transparent background. Dark mode may make this difficult to see. Please see below:

![image](https://user-images.githubusercontent.com/78788406/140649298-36b96315-fa9b-4353-9185-7222a4f8fba7.png)

Please advise?

Reply:
Can you add a white background?